Source/WarpX.cpp

```diff
@@ -925,6 +911,25 @@ WarpX::ReadParameters ()
                                " some numerical artifact will be present at the interface between coarse and fine patch."
                                "\nWe recommend setting algo.particle_shape = 1 in order to avoid this issue");
             }
+            
+            // default sort interval for particles if species or lasers vector is not empty 
+#ifdef AMREX_USE_GPU
+            std::vector<std::string>sort_intervals_string_vec = {"4"};
+#else
+            std::vector<std::string> sort_intervals_string_vec = {"-1"};
+#endif
+        } else {
+            // default sort interval for particles is -1 if species/laser vector is empty
+            std::vector<std::string>sort_intervals_string_vec = {"-1"};
+        }
+        pp_warpx.queryarr("sort_intervals", sort_intervals_string_vec);
```

Comment:
This might be missing, looking quickly at the CI build logs:
```suggestion
        amrex::ParmParse pp_warpx("warpx");
        pp_warpx.queryarr("sort_intervals", sort_intervals_string_vec);
```

Reply:
@RevathiJambunathan About this PR, it might be a good idea to rebase on `development` now, besides committing this small fix above.

Reply:
Thank Edoardo for catching this!